suite-native/module-device-settings/src/components/DeviceFirmwareCard.tsx

```diff
@@ -66,6 +80,16 @@ export const DeviceFirmwareCard = () => {
                         />
                     ),
                     variant: 'info',
+                    rightButton: (
+                        <Button
+                            colorScheme="blueBold"
```

Comment:
The size is not right:

```suggestion
                            colorScheme="blueBold"
                            size="small"
```